challenges/support/README.md

```diff
@@ -0,0 +1,66 @@
+# Summary
+
+The Solutions and Support Engineer sysadmin challenge is a simulated troubleshooting session where several faults have been created in a
+sample environment requiring the candidate to investigate and solve the problems. 
+
+
+> **Note:** For SREs/Devops without a backend development background, the on-call sysadmin challenges are used, but instead of remote hands, shell access to the nodes will be provided by provisioning a public ssh key.
+
+# Rationale
+
+This exercise has two goals.
+
+- It helps us understand what to expect from you as an engineer,  how you reason about production systems and investigate faults.
+- It helps you get a feel for what it would be like to work at Teleport, as this exercise aims to simulate some of our day-as-usual and expose you to the type of work we're doing here.
+
+We believe this technique is better and is more fun compared to whiteboard/quiz interviews so common in the industry. It's not without the downsides as it could take longer than traditional interviews. [Some of the best teams use coding challenges.](https://sockpuppet.org/blog/2015/03/06/the-hiring-post/)
+
+We appreciate your time and are looking forward to hacking on this challenge together.
+
+# Requirements
+The challenge is set up as a remote troubleshooting session. The candidate and Teleport engineers in similar roles will join a live troubleshooting session via zoom screen sharing. The candidate will be required to solve several faults created in a simulated customer environment. 
```

Comment:
In the previous section, we refer to the candidate as _you_, but here we're using third-person instead. I'd suggest writing it like this:

```suggestion
The challenge is set up as a remote troubleshooting session.
Several Teleport engineers will join you in a Zoom call where you will
work to solve several faults in a simulated customer environment.
```